### PR TITLE
Main screen

### PR DESCRIPTION
**Pull request by avdgupta**

### CODE REVIEW

File: src/containers/Home/dashboardHome.style.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { styled } from "@mui/material/styles";
+import MuiAccordionDetails from "@mui/material/AccordionDetails";
+
+export const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
```
Comment (aritraibm) [inline src/containers/Home/dashboardHome.style.ts]: If you want, you can use the Project Details screen structure for accordion and others, it's already created, which may give us the same look and feel for entire the application, thanks

File: src/containers/Home/dashboardHome.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    const handleChange =
+        (panel: string) =>
+        (event: React.SyntheticEvent, newExpanded: boolean) => {
+            setExpanded(newExpanded ? panel : false);
+        };
+
+    return (
+        <div style={DivStyle}>
+            <Accordion
+                expanded={expanded === "panel1"}
```
Comment (aritraibm) [inline src/containers/Home/dashboardHome.tsx]: Please use the proper naming convention for the accordion name.

https://google.github.io/styleguide/tsguide.html#syntax

File: src/containers/Home/dashboardHome.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    AccordionSummaryProps,
+} from "@mui/material/AccordionSummary";
+import Typography from "@mui/material/Typography";
+import { UIConstants } from "@constants/UIConstants";
+import { AccordionDetails, DivStyle } from "./dashboardHome.style";
+
+const Accordion = styled((props: AccordionProps) => (
+    <MuiAccordion disableGutters elevation={0} square {...props} />
+))(({ theme }) => ({
+    border: `1px solid ${theme.palette.divider}`,
```
Comment (aritraibm) [inline src/containers/Home/dashboardHome.tsx]: Move this into style if possible, thanks

File: src/containers/Home/dashboardHome.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    },
+}));
+
+const AccordionSummary = styled((props: AccordionSummaryProps) => (
+    <MuiAccordionSummary
+        expandIcon={<ArrowForwardIosSharpIcon sx={{ fontSize: "0.9rem" }} />}
+        {...props}
+    />
+))(({ theme }) => ({
+    backgroundColor:
```
Comment (aritraibm) [inline src/containers/Home/dashboardHome.tsx]: Can we please move all the styles to their respective style files, thanks

**Review by aritraibm — DISMISSED** [review]

**Review by PramodaM4444 — APPROVED** [review]

**Review by ShubhamAgrawa — APPROVED** [review]

**Pull request merged**